packages/plugin-core/src/survey.ts

```diff
@@ -0,0 +1,219 @@
+  }
+}
+
+export class PriorToolsSurvey extends DendronSurvey {
+  async onAnswer(results: vscode.QuickPickItem[]) {
+    let maybeOtherResult: string | undefined;
+    if (results.some(result => result.label === "Other" )) {
+      maybeOtherResult = await vscode.window.showInputBox({
+        ignoreFocusOut: true,
+        placeHolder: "Type anything that applies.",
+        prompt: "You have checked \"Other\". Please describe what other tools you have used.",
+        title: "Are you coming from an existing tool? - Other",
+      });
+    }
+    AnalyticsUtils.track(SurveyEvents.PriorToolsAnswered, {
+      results: results.map((result) => result.label),
+      other: maybeOtherResult,
+    });
+  }
+
+  onReject() {
+    AnalyticsUtils.track(SurveyEvents.PriorToolsRejected);
+  }
+
+  static create() {
+    const title = "Are you coming from an existing tool?";
+    const choices = [
+      { label: "No" },
+      { label: "Foam" },
+      { label: "Roam" },
+      { label: "Logseq" },
+      { label: "Notion" },
+      { label: "OneNote" },
+      { label: "Obsidian" },
+      { label: "Evernote" },
+      { label: "Google Keep" },
+      { label: "Other" },
+    ];
+    return new PriorToolsSurvey({ title, choices, canPickMany: true });
+  }
+}
+
+export class SurveyUtils {
+  /**
+   * Flip a coin to randomly prompt initial survey.
+   * Asks three questions about background, use case, and prior tools used.
+   * @param forcePrompt skip flipping a coin and force prompting.
+   */
+  static async maybePromptInitialSurvey(forcePrompt?: boolean) {
+    const shouldPrompt = forcePrompt ? true : !!Math.floor(Math.random() * 2);
+    if (shouldPrompt) {
+      AnalyticsUtils.track(SurveyEvents.InitialSurveyPrompted);
+      vscode.window
+        .showInformationMessage(
+          "Welcome to Dendron!",
+          { modal: true , detail: "Would you like to tell us a bit about yourself? This info will be used to provide a better onboarding experience. This will take less than a minute to complete."},
+          { title: "Proceed" }
+        )
+        .then(async (resp) => {
+          if (resp?.title === "Proceed") {
+            const backgroundSurvey = BackgroundSurvey.create();
+            const useCaseSurvey = UseCaseSurvey.create();
+            const priorToolSurvey = PriorToolsSurvey.create();
+
+            const backgroundResults = await backgroundSurvey.show(1, 3);
+            const useCaseResults = await useCaseSurvey.show(2, 3);
+            const priorToolsResults = await priorToolSurvey.show(3, 3);
+
+            const answerCount = [
+              backgroundResults,
+              useCaseResults,
+              priorToolsResults,
+            ].filter((value) => !_.isUndefined(value)).length;
+            AnalyticsUtils.track(SurveyEvents.InitialSurveyAccepted, {
+              answerCount,
+            });
+            vscode.window.showInformationMessage("Survey submitted.");
```

Comment:
add a bit more character in this message -> Survey submitted! Thanks for helping us make Dendron better 🌱

Reply:
Changed as suggested (and some more in other places)